offchain-modules/packages/scripts/src/cardano-integ.ts

```diff
@@ -129,6 +130,7 @@ function getBridgeAddr(config: MultisigConfig): string {
 }
 
 async function main() {
+  // eslint-disable-next-line
```

Comment:
Is this necessary or something done for development?

Reply:
Mistake, I will take out